docs/internal-developers/testing/releases/840.md

```diff
@@ -0,0 +1,138 @@
+5. Now go to your Edit Page & Edit Template pages and enable "'Apply filters' button".
+6. On the frontends again, re-select your filter options and check that they are not applied immediately until you click "Apply"
+7. Check that when this block is used with the PHP template & the All Products block that regardless of whether the "Apply Filters" button is active, that clicking "Reset" immediately clears the selected filter options
+
+| Before | After |
+| ------ | ----- |
+|    <img width="509" alt="Screenshot 2022-08-17 at 12 33 57" src="https://user-images.githubusercontent.com/8639742/185108568-1c6875e8-10a1-4070-919d-876c42af8a89.png">    |   <img width="555" alt="Screenshot 2022-08-17 at 12 32 28" src="https://user-images.githubusercontent.com/8639742/185410884-45952fdd-14fe-4784-9ba6-c58c248630ad.png">    |
+
+### Update the design of the Filter Products by Price block. ([6877](https://github.com/woocommerce/woocommerce-blocks/pull/6877))
+
+1. Add `Filter Products by Price` block to a page.
+2. Verify that the `Inline input fields` only shows when the `Price Range` is Editable.
+3. Toggle Inline input fields, see the block in the editor updates accordingly.
+4. Toggle Filter button, see the Reset and Apply button.
+5. Add the All Products block then save the page.
+6. On the front end, see the new style applied as in the editor.
+7. Change the price using the slider then click the Apply button.
+8. See the filter works as expected.
+9. Click on the Reset button, see the price filter is reset. (Even if the filter button is enabled, clicking on the reset button should reset the filter query).
+10. Tested again with PHP templates, ensure no regression.
+
+<img width="1256" alt="image" src="https://user-images.githubusercontent.com/5423135/184097877-5210b4c6-907b-404f-bcf3-c7d83c02a563.png">
+
+### Allow making the Cart/Checkout block page the default one from within the editor. ([6867](https://github.com/woocommerce/woocommerce-blocks/pull/6867))
+
+To test this feature you need the latest trunk of WooCommerce, or WooCommerce 6.9.0. For older versions of WC (pre 6.9.0), the notice would show the old text:
+
+<img width="297" alt="image" src="https://user-images.githubusercontent.com/6165348/184404663-11cac5f1-1729-4782-a39f-ee5f101a78a7.png">
+
+1. Make sure you have a Cart and Checkout pages in WooCommerce -> Settings -> Advanced.
+2. Those pages urls can be `/cart` and `/checkout`.
+3. Open a new page, give it a title, insert Cart block.
+4. Select Cart block, you should see the notice, select an inner block, you should see the notice as well.
+5. Click on the notice, it will load for a couple of seconds, once it's green, visit your new page.
+6. Make sure it's on the previous url, so `/cart`.
+7. Go back to settings, make sure your new page is that value there.
+8. Do the same thing for Checkout block.
+9. Open a new page, add Cart block, save the page and publish it.
```

Comment:
Shouldn't this mention _Checkout block_ instead of _Cart Block_?

Also, it is confusing since it says to save and publish whereas the previous instruction for Cart block didn't. If you do it and stay on the page, the update button is presented but if you leave and come back to edit, it is not.